Description: No longer generates an unneeded blank at the beginning of a line
Origin: vendor
Bug-Debian: https://bugs.debian.org/255631
Forwarded: no
Author: Gunnar Wolf <gwolf@debian.org>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2019-12-22

--- a/lib/HTML/Clean.pm
+++ b/lib/HTML/Clean.pm
@@ -428,7 +428,7 @@ sub strip {
     $$h =~ s,[\r\n]+,\n,sg; # Carriage/LF -> LF
     $$h =~ s,\s+\n,\n,sg;   # empty line
     $$h =~ s,\n\s+<,\n<,sg; # space before tag
-    $$h =~ s,\n\s+,\n ,sg;  # other spaces
+    $$h =~ s,\n\s+,\n,sg;   # other spaces
 
     $$h =~ s,>\n\s*<,><,sg; # LF/spaces between tags..
 
